### ISSUE TITLE
alpaca fails to install on widows 8.1

### ISSUE THREAD

**Issue by user111111** [gitana/alpaca#290]
When I run "npm install alpaca" the operation fails. The extract of the npm log file:

---------------------------------
13863 error Windows_NT 6.3.9600
13864 error argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "alpaca"
13865 error node v0.12.7
13866 error npm  v2.11.3
13867 error code ELIFECYCLE
13868 error alpaca@1.5.11 postinstall: `bower install`
13868 error Exit status 1
13869 error Failed at the alpaca@1.5.11 postinstall script 'bower install'.
13869 error This is most likely a problem with the alpaca package,
13869 error not with npm itself.
13869 error Tell the author that this fails on your system:
13869 error     bower install
---------------------------------

What should be done?

**Comment by basvandenheuvel** [comment]
Download latest release from github?

**Comment by uzquiano** [comment]
I don't know about Windows 8.1 but I just tried it on the latest Windows 10 and it works just fine.  I suppose if someone wants to try it specifically on Windows 8.1, they might offer some notes.  But it definitely should work.  

Note that some NPM dependency modules may have varying requirements depending on your build platform.  I put a note in the README about the python dependency.  In the past, I've also had to have make available for some more obscure node modules and that can involve installing Visual Studio.

However, as far as I'm aware, the module set required by Alpaca is a pretty standard one.  There shouldn't be any curveballs.
